calico-enterprise/_includes/content/_rule.mdx

```diff
@@ -12,6 +12,7 @@ are executed in order.
 | ipVersion   | Positive IP version match.                 | `4`, `6`                                                     | integer                       |         |
 | source      | Source match parameters.                   |                                                              | [EntityRule](#entityrule)     |         |
 | destination | Destination match parameters.              |                                                              | [EntityRule](#entityrule)     |         |
+| http        | Match HTTP request parameters. Application layer policy must be enabled to use this field.|               | [HTTPMatch](#httpmatch)       |         |
```

Comment:
> Application layer policy must be enabled to use this field.

Nit: might be too long? I feel you might want to reserve this information where HTTPMatch is explained / where the anchor is linking

Reply:
It's looking good in the table :). Do you want me to move it?